Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/Orbifold_Tutte_parameterizer_3.h

```diff
@@ -790,43 +775,34 @@ class Orbifold_Tutte_parameterizer_3
                            VertexIndexMap vimap,
                            Matrix& M) const
   {
-    const PPM ppmap = get(vertex_point, mesh);
+    const PPM pmap = get(vertex_point, mesh);
+    for (const halfedge_descriptor hd : halfedges(mesh)) {
 
-    // not exactly sure which cotan weights should be used:
-    // 0.5 (cot a + cot b) ? 1/T1 cot a + 1/T2 cot b ? 1/Vor(i) (cot a + cot b?)
-    // Comparing to the matlab code, the basic Cotangent_weight gives the same results.
-    typedef CGAL::internal::Cotangent_weight<SeamMesh>                      Cotan_weights;
-//    typedef CGAL::internal::Cotangent_weight_with_triangle_area<SeamMesh>   Cotan_weights;
-
-    Cotan_weights cotan_weight_calculator(mesh, ppmap);
-
-    for(halfedge_descriptor hd : halfedges(mesh)) {
       const vertex_descriptor vi = source(hd, mesh);
       const vertex_descriptor vj = target(hd, mesh);
       const int i = get(vimap, vi);
       const int j = get(vimap, vj);
 
-      if(i > j)
-        continue;
-
-      // times 2 because Cotangent_weight returns 1/2 (cot alpha + cot beta)...
-      const NT w_ij = 2 * cotan_weight_calculator(hd);
+      if (i > j) continue;
+      const CGAL::Weights::internal::
```

Comment:
It seems not logical that we have to use a function from `internal`  here.   The whole idea of it is that it does only gets  used by the weights package itself.

Reply:
The internal is removed and all wrappers are renamed and moved to the corresponding weight headers.